.gitignore

```diff
@@ -1,3 +1,4 @@
 /*.js
 node_modules/
 .DS_Store
+hathora.yml
```

Comment:
what was the reason for adding this?

Reply:
I ended up with a `hathora.yml` in my repo through the process of testing and almost checked it in - I figured that this wouldn't hurt and might prevent a mistake in the future!

Reply:
i think this could cause issues with the example apps in this repo since they have a `hathora.yml` file which is supposed to be checked in

Reply:
If I scoped it to the project root would that be acceptable?

Reply:
i would prefer if we just got rid of it